### PR TITLE
update elan URL

### PR DESCRIPTION
**Pull request by bryangingechen**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -38,7 +38,7 @@ jobs:
       if: runner.os == 'macOS'
     - name: Install elan
       run: |
-          curl https://raw.githubusercontent.com/Kha/elan/master/elan-init.sh -sSf | sh -s -- --default-toolchain leanprover-community/lean:3.5.1 -y 
+          curl https://raw.githubusercontent.com/leanprover/elan/master/elan-init.sh -sSf | sh -s -- --default-toolchain leanprover-community/lean:3.5.1 -y
```
Comment (bryangingechen) [inline .github/workflows/ci.yml:41]: This hardcodes Lean 3.5.1; I guess it hasn't hurt up to now, but this might be something we want to change.

**Comment by Julian** [comment]
lgtm if my vote counts

**Comment by PatrickMassot** [comment]
I completely missed that PR. I have no idea where the hardcoded 3.5.1 comes from. What happens if we simply don't specify a default toolchain?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (bryangingechen) [reply .github/workflows/ci.yml:41]:
```suggestion
          curl https://raw.githubusercontent.com/leanprover/elan/master/elan-init.sh -sSf | sh -s -- --default-toolchain none -y
```

**Comment by bryangingechen** [comment]
I just changed the default toolchain to `none` to follow mathlib's CI; let's see if it works.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -38,7 +38,7 @@ jobs:
       if: runner.os == 'macOS'
     - name: Install elan
       run: |
-          curl https://raw.githubusercontent.com/Kha/elan/master/elan-init.sh -sSf | sh -s -- --default-toolchain leanprover-community/lean:3.5.1 -y 
+          curl https://raw.githubusercontent.com/leanprover/elan/master/elan-init.sh -sSf | sh -s -- --default-toolchain none -y
```
Comment (bryangingechen) [inline .github/workflows/ci.yml:41]:
```suggestion
          curl https://raw.githubusercontent.com/leanprover/elan/master/elan-init.sh -sSf | sh -s -- -y
```

**Comment by bryangingechen** [comment]
OK, so `none` doesn't work, but not specifying a default works after all!

**Review by eric-wieser — APPROVED** [review]
CI passes, so this looks good to me

**Pull request closed without merging**